### DISCUSSION TITLE
Changes to .env.local are not loaded even after restarting

### DISCUSSION THREAD

**Discussion by mattpodolak** [vercel/next.js#22688] [Help]
**What version of Next.js are you using?**

10.0.3

**What version of Node.js are you using?**

14.5.0

**What browser are you using?**

Chrome

**What operating system are you using?**

Windows

**How are you deploying your application?**

local

**Describe the Bug**

Adding environment variables to .env.local are not being loaded, however, the current env variables are still accessible

**Expected Behavior**

It should load newly added environment variables

**To Reproduce**

Not sure, changes to .env.local randomly stopped loading into next.js

**Comment by ahmedjahmi** [comment]
having a simlar issue, not sure what's wrong. if you've found a solution, please reply. @mattpodolak

**Comment by leerob** [comment]
Are you restarting the server? A message is outputted saying "please restart your server to load the new environment variables".
